Quaver.Shared/Screens/Edit/EditScreen.cs

```diff
@@ -374,13 +381,16 @@ private void SetAudioTrack(IAudioTrack track)
                 Track = track;
                 Track.ApplyRate(ConfigManager.Pitched?.Value ?? true);
 
-                lock (AudioEngine.Track)
+                if (AudioEngine.Track != null)
```

Comment:
Why have you moved the lock inside?

Reply:
Yes, because it crashed in Visual Tests and Swan told me to make that change